src/tests/GC/Stress/testmix_gc.config

```diff
@@ -2,7 +2,7 @@
 <Test xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
             id="NightlyRun" 
             maximumTestRuns="-1" 
-            maximumExecutionTime="15:15:00" 
+            maximumExecutionTime="5:15:00" 
```

Comment:
Was this change just for your testing?

Reply:
yes.. I shouldn't have checked this in.. and I'll revert it. I just noticed that I'll need to update the PR anyway (because the merge conflicts are too complicated to resolve online). so I'll also include changes that you suggested in other comments.